app/controllers/application_controller.rb

```diff
@@ -83,4 +83,8 @@ def track_get_requests
 
     analytics.track_event("GET request for #{controller_name}##{action_name}")
   end
+
+  def create_user_event(event_type, user = current_user)
+    Event.create(user_id: user.id, event_type: event_type)
+  end
```

Comment:
Add a test for this to make sure that calling `create_user_event` creates an event with the expected values?